query-service-api/src/main/proto/request.proto

```diff
@@ -73,6 +74,12 @@ message ColumnIdentifier {
   string alias = 2;
 }
 
+message AttributeExpression {
+  string attributeId = 1;
+  string subpath = 2;
+  string alias = 3;
```

Comment:
```suggestion
  optional string alias = 3;
```

Reply:
Done.
Isn't it optional by default ?

Reply:
Any field can be omitted, but optional allows for field presence checking. Any primitive field that we expect to be omitted in a reasonable operation, and want the consumer to _know_ we omitted it should be marked optional. Explained in detail at https://github.com/protocolbuffers/protobuf/blob/master/docs/field_presence.md

Reply:
resolved